Sources/Clappr/Classes/Enum/Event.swift

```diff
@@ -67,4 +67,5 @@ public enum Event: String, CaseIterable {
     case hideDrawerPlugin = "Clappr:hideDrawerPlugin"
     case willHideDrawerPlugin = "Clappr:willHideDrawerPlugin"
     case didHideDrawerPlugin = "Clappr:didHideDrawerPlugin"
+    case didDragDrawer = "Clappr:didDragDrawer"
```

Comment:
Do we want to make it an internal event? I'm not sure if the apps will want to listen to this event 🤷‍♂

Reply:
Done.